src/components/project-list-card.astro

```diff
@@ -52,6 +53,7 @@ const { data } = Astro.props;
   dl {
     display: grid;
     grid-template-columns: 1fr 1fr;
+    text-align: end;
```

Comment:
Is this necessary? Looks a bit stretched on wider displays.

Reply:
Oh that was probably a thing to make the text stick in an earlier iteration, can remove this now.